src/kernel/heap.zig

```diff
@@ -182,16 +182,18 @@ const FreeListAllocator = struct {
     /// Arguments:
     ///     IN allocator: *std.Allocator - The allocator to resize within.
     ///     IN old_mem: []u8 - The buffer to resize.
+    ///     IN mem_align: u29 - The original alignment for old_mem.
```

Comment:
Maybe `old_align` would be better

Reply:
Sure. This name was copied straight from std lib.